src/main/java/net/atos/zac/app/zaken/converter/RESTZaakConverter.java

```diff
@@ -142,7 +142,8 @@ public RESTZaak convert(final Zaak zaak, final Status status, final Statustype s
         restZaak.resultaat = zaakResultaatConverter.convert(zaak.getResultaat());
 
         restZaak.isOpgeschort = zaak.isOpgeschort();
-        if (zaak.isOpgeschort()) {
+        restZaak.isHervat = !zaak.getOpschorting().getReden().isEmpty() && !zaak.isOpgeschort();
```

Comment:
Gebruik hier StringUtils.isNotEmpty(zaak.getOpschorting().getReden()). Want het zou ook kunnen zijn dat reden null is.

Reply:
Opgelost in opvolgende commit